cypress.config.js

```diff
@@ -1,8 +1,26 @@
 const { defineConfig } = require('cypress')
+const { faker } = require('@faker-js/faker');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl:'https://demoqa.com/automation-practice-form',
     setupNodeEvents(on, config) {
+      on("task", {
+        generateUser() {
+          let randomIndex = Math.floor(Math.random * 2);
```

Comment:
Replace 'Math.random * 2' with 'Math.random() * 2' to correctly generate a random index. 🧮

Reply:
done